internal/core/infrastructure/feeder/kraken/service.go

```diff
@@ -168,7 +216,7 @@ func (k *service) parseFeed(msg []byte) ports.PriceFeed {
 	return &priceFeed{
 		market: mkt,
 		price: &price{
-			basePrice:  basePrice.String(),
+			basePrice:  basePrice.StringFixed(8),
```

Comment:
is it possible to have a float with `0.xxx` here? there fore the stirng fixed shoudl be 10?

Reply:
only 3 decimals seems not enough to me. For BTC/USD ,for example, the base price (hence 1/(BTCUSD price) expresses a BTC unit, therefore 8 decimal precision.

Reply:
8 decimals plus the dot and the base unit, therefore `StringFixed(10)` ?